science/2022-03-03-science-blog-5/index_de.md

```diff
@@ -0,0 +1,898 @@
+1.5
+</td>
+<td style="text-align:left;width: 11cm; ">
+Unterstützung des europäischen Corona-App-Gateways, freiwillige Symptomerfassung, Verbesserung der Textverständlichkeit
+</td>
+</tr>
+<tr>
+<td style="text-align:left;width: 3cm; ">
+1.6
+</td>
+<td style="text-align:left;width: 11cm; ">
+Detaillierte Erklärung bei Wechsel des Risikostatus
+</td>
+</tr>
+<tr>
+<td style="text-align:left;width: 3cm; ">
+1.7
+</td>
+<td style="text-align:left;width: 11cm; ">
+Risikoüberprüfung kann mehrmals täglich durchgeführt werden, Erinnerungsfunktion für die Teilung positiver Testergebnisse via CWA
+</td>
+</tr>
+<tr>
+<td style="text-align:left;width: 3cm; ">
+1.9
+</td>
+<td style="text-align:left;width: 11cm; ">
+Umstellung auf G/A-ENF Version 2, Verbesserung der Risikobewertung, Verbesserung vom Ablauf des Einscannens der Testergebnisse bis zum Teilen der Diagnoseschlüssel
+</td>
+</tr>
+<tr>
+<td style="text-align:left;width: 3cm; ">
+1.10
+</td>
+<td style="text-align:left;width: 11cm; ">
+Kontakttagebuch
+</td>
+</tr>
+<tr>
+<td style="text-align:left;width: 3cm; ">
+1.11
+</td>
+<td style="text-align:left;width: 11cm; ">
+Kennzahlen zum Infektionsgeschehen
+</td>
+</tr>
+<tr>
+<td style="text-align:left;width: 3cm; ">
+1.12
+</td>
+<td style="text-align:left;width: 11cm; ">
+Erweiterung des Kontakttagebuchs (Begegnungshistorie), Unterstützung von den iPhones 5s, 6 und 6 Plus
+</td>
+</tr>
+<tr>
+<td style="text-align:left;width: 3cm; ">
+1.13
+</td>
+<td style="text-align:left;width: 11cm; ">
+Freiwillige Datenspende (Privacy-Preserving-Analytics (PPA))
+</td>
+</tr>
+<tr>
+<td style="text-align:left;width: 3cm; ">
+1.14
+</td>
+<td style="text-align:left;width: 11cm; ">
+Erweiterung des Kontakttagebuchs (u.a. Pop-Up-Menü), Erweiterung der freiwilligen Datenspende (neue Datenpunkte)
+</td>
+</tr>
+<tr>
+<td style="text-align:left;width: 3cm; ">
+1.15
+</td>
+<td style="text-align:left;width: 11cm; ">
+Länderübergreifende Risikoübermittlung (Schweiz), Veränderung der Risikokarte
+</td>
+</tr>
+<tr>
+<td style="text-align:left;width: 3cm; ">
+2.0
+</td>
+<td style="text-align:left;width: 11cm; ">
+Eventregistrierung (Check-in)
+</td>
+</tr>
+<tr>
+<td style="text-align:left;width: 3cm; ">
+2.1
+</td>
+<td style="text-align:left;width: 11cm; ">
+Integration von Antigen-Schnelltests
+</td>
+</tr>
+<tr>
+<td style="text-align:left;width: 3cm; ">
+2.2
+</td>
+<td style="text-align:left;width: 11cm; ">
+Antigen-Schnelltest-Profil und Fehlerberichte
+</td>
+</tr>
+<tr>
+<td style="text-align:left;width: 3cm; ">
+2.3
+</td>
+<td style="text-align:left;width: 11cm; ">
+Integration des digitalen Impfnachweises
+</td>
+</tr>
+<tr>
+<td style="text-align:left;width: 3cm; ">
+2.4
+</td>
+<td style="text-align:left;width: 11cm; ">
+Testzertifikate für Antigen-Schnelltests und PCR-Tests, digitales EU-Testzertifikat, Erweiterung Kontakttagebuchs (automatisierter Eintrag von Testergebnissen), Erweiterung der freiwilligen Datenspende (neue Datenpunkte)
```

Comment:
"Kontakttagebuchs" => "Kontakt-Tagebuchs"

Reply:
Fixed in the latest commit, thanks!